### ISSUE TITLE
feat: Add third-party integration section

### ISSUE THREAD

**Issue by afontcu** [vuejs/vue-test-utils-next-docs#30]
https://github.com/vuejs/vue-test-utils-next-docs/blob/master/src/guide/third-party.md

**Comment by lmiller1990** [comment]
I think we should use Vuetify for this probably, it is the main third party lib people have trouble with.

I think we should recommend `mount` for Vuetify. Vuetify also use `mount` for their unit tests, maybe we could link to some of the actual Vuetify tests for examples of "real world tests with Vuetify".

**Comment by afontcu** [comment]
+1 at using Vuetify, it is a common source of issues (people want to stay back from rendering Vuetify components, but then run into issues with stubbed slots and so on). Is there a public release of vuetify for Vue 3? Couldn't find it.

**Comment by lmiller1990** [comment]
No, I don't think so. There is the `next` branch: https://github.com/vuetifyjs/vuetify/tree/next

No rush, was just putting some thoughts here for when we get to it 👍
